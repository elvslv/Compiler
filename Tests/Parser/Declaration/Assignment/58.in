type trec = record x: integer; y: float; end;
var arr = array[1..10] of trec;
x: integer;
begin
arr[x].y := 1234.5;
end.