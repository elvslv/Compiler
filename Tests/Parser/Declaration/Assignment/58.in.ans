type TREC = record 
    var X : INTEGER;
    var Y : REAL;
end;
var ARR : array [const 1..const 10] of TREC;
var X : INTEGER;

:=             
|              
|___.          
|   |          
|   |___[]     
|   |   |      
|   |   |___ARR
|   |   |      
|   |   |___X  
|   |          
|   |___Y      
|              
|___1234.5     
