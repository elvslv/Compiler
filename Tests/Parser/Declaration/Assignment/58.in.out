type TREC = record 
    var X : INTEGER;
    var Y : FLOAT;
end;
9 2 Invalid lexem in var block